### PR TITLE
Feature/share permissions via attributes

### PR DESCRIPTION
**Pull request by DeepDiver1975**
requires https://github.com/owncloud/core/pull/36024 for the test which sets share permissions via attributes

### CODE REVIEW

File: src/shareManagement.js [hunk 1]
```diff
@@ -65,27 +65,26 @@ class Shares {
...
 
-    return new Promise((resolve, reject) => {
-      this.helpers._makeOCSrequest('POST', this.helpers.OCS_SERVICE_SHARE, 'shares', postData)
-        .then(data => {
-          //          data.body = utf8.decode(data.body)
-          const shareDetails = parser.xml2js(data.body).ocs.data
-          const share = new ShareInfo(shareDetails)
+    return this.helpers._makeOCSrequest('POST', this.helpers.OCS_SERVICE_SHARE, 'shares', postData)
+      .then(data => {
+        //          data.body = utf8.decode(data.body)
```
Comment (PVince81) [inline src/shareManagement.js]: remove ?

File: yarn.lock [hunk 2]
```diff
@@ -731,9 +731,9 @@
   integrity sha512-tHq6qdbT9U1IRSGf14CL0pUlULksvY9OZ+5eEgl1N7t+OA3tGvNpxJCzuKQlsNgCVwbAs670L1vcVQi8j9HjnA==
 
 "@types/node@*":
-  version "12.7.1"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-12.7.1.tgz#3b5c3a26393c19b400844ac422bd0f631a94d69d"
-  integrity sha512-aK9jxMypeSrhiYofWWBf/T7O+KwaiAHzM4sveCdWPn71lzUSMimRnKzhXDKfKwV1kWoBo2P1aGgaIYGLf9/ljw==
+  version "12.7.2"
```
Comment (PVince81) [inline yarn.lock]: was this deps update intended to be in this PR ? I believe there is already another PR from dependabot for that

File: .drone.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@ steps:
   pull: always
   image: owncloudci/core
   settings:
-    GIT_REFERENCE: "master"
+    GIT_REFERENCE: "feature/poc-roles-api"
```
Comment (PVince81) [inline .drone.yml]: I've merged the core PR, you can revert this

**Comment by DeepDiver1975** [comment]
The failing unit tests are unrelated ... needs own PR

**Pull request merged**